CMakeLists.txt

```diff
@@ -20,7 +20,7 @@
 #
 ############################################################################
 
-cmake_minimum_required(VERSION 3.22)
+cmake_minimum_required(VERSION 3.5)
```

Comment:
You're changing it from 3.22 to 3.5 here though? (i.e. a downgrade)

Reply:
because I confused patches, sorry for the noise

Reply:
np :)